src/System.DirectoryServices.AccountManagement/tests/UserPrincipalTest.cs

```diff
@@ -29,33 +29,5 @@ public void UserPrincipalConstructorTest()
             UserPrincipal user = new UserPrincipal(DomainContext);
             user.Dispose();
         }
-
-        public void ComputedUACCheck()
```

Comment:
I am not sure if it is okay to remove this test.

Reply:
I don't think we run it, so it should be fine.

Reply:
It was never marked as a test. I (slightly arbitrarily) decided to delete it rather than fix it as it seemed hacky. At some point we in dotnet/runtime might want to go through all the other old DS tests that never run and either enable them or delete them.

Reply:
ohhh, make sense